exporter/exporterhelper/metricshelper_test.go

```diff
@@ -87,6 +91,33 @@ func TestMetricsExporter_WithSpan_ReturnError(t *testing.T) {
 	checkWrapSpanForMetricsExporter(t, te, want, 0)
 }
 
+func TestMetricsExporter_WithShutdown(t *testing.T) {
+	shutdownCalled := false
+	shutdown := func() error { shutdownCalled = true; return nil }
+	me, err := NewMetricsExporter(fakeExporterName, newPushMetricsData(0, nil), WithShutdown(shutdown))
+	if err != nil {
```

Comment:
nit: use the testify assert and require for new tests

Reply:
Yeah I was trying to be consistent in this file. I'll update this and I created this jira to migrate to testify/assert instead of this t.Fatal() pattern
https://github.com/open-telemetry/opentelemetry-service/issues/266